Task1/Program.cs

```diff
@@ -10,7 +10,268 @@ class Program
     {
         static void Main(string[] args)
         {
-            
+            //Factorial(5);
+            //SumBetweenNumbers(3, 6);
+            //OddValuesBetweenNumbers(5, 12);
+            //CounOfEvenInNumber(34566510);
+            //MeanOfNumbers();
+            //DaysOfRunning();
+            //Square(30);
+            //ReverseOfNumber();
+            //PowOfNumber(5, 2);
+            //PropertyOfNumber();
+            //Multiplication(-5, -3);
+            //DeletDigit();
+            MaxDigit(-16759);
+            Console.ReadKey();
+        }
+   
+        public static void Factorial(int number)
+        {
+            int factorial = 1;
+            while (number > 0)
+            {
+                factorial = factorial * number;
+                number--;
+            }
+            Console.WriteLine($"Factorial = {factorial}");
+        }
+
+        public static void SumBetweenNumbers(int number1, int number2)
+        {
+            int sum = 0;
+            for (int i = number1+1; i < number2; i++)
+            {
+                sum += i;
+            }
+            Console.WriteLine($"Sum of numbers between {number1} and {number2} = {sum}");
+        }
+
+        public static void OddValuesBetweenNumbers(int number1, int number2)
+        {
+            String odd = "";
```

Comment:
Please use alias string instead of String.
Also you already know that we should use string.Empty instead of "".